src/alert/alert.scss

```diff
@@ -15,28 +15,29 @@
     color: t(iui-text-color);
```

Comment:
Done but link looks bad in dark theme
![image](https://user-images.githubusercontent.com/9084735/136244648-0304eaed-4ba6-426a-93f8-bca03e59d98a.png)

Reply:
We also don't support non-status alerts in React, so that mockup would need to be constructed through plain html

Reply:
The hyperlink part I agree does look bad, maybe we should also color that to our typical hyperlink color?

Would we be able to offer this non-status alert in React?

Reply:
I'll create a separate PR for default link styling.

We can offer non-status Alerts if we want, but we do set the "default" to `informational`, so it would be kinda weird to have to set status to `none` or something.